.gitignore

```
@@ -17,4 +17,7 @@ yarn-error.log*
 
 hop.sh
 1
-yarn.lock
\ No newline at end of file
+yarn.lock
+
+airdrop-reopen.sublime-workspace
```

suggestion: ignore all sublime workspaces/projects in gitignore

```suggestion
*.sublime-workspace
*.sublime-project
```